tools/Azure.Mcp.Tools.AppService/src/Options/AppServiceOptionDefinitions.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.Mcp.Tools.AppService.Options;
+
+public static class AppServiceOptionDefinitions
+{
+    public const string AppName = "app";
+    public const string DatabaseType = "database-type";
+    public const string DatabaseServer = "database-server";
+    public const string DatabaseName = "database";
+    public const string ConnectionString = "connection-string";
+
+    public static readonly Option<string> AppServiceName = new(
```

Comment:
all of these need explicit description property

Reply:
added